airbyte-integrations/connectors/source-pinterest/source_pinterest/spec.json

```diff
@@ -0,0 +1,50 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/pinterest",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Pinterest Spec",
+    "type": "object",
+    "required": ["client_id", "client_secret", "refresh_token", "start_date"],
+    "additionalProperties": false,
+    "properties": {
```

Comment:
we should create a ticket to support oauth for this connector

Reply:
Created https://github.com/airbytehq/airbyte/issues/7754